kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-update3-d1-JRF-v2-ds.yaml

```diff
@@ -122,8 +122,17 @@ spec:
   # roll of your domain's WebLogic Server pods.
   restartVersion: '1'
 
+  # Changes to this field cause the operator to repeat its introspection of the
+  #  WebLogic domain configuration.
+  introspectVersion: '1'
+
   configuration:
 
+    # Operator will attempt to use the online update to the running domain without
+    # restarting the servers if only changes made are updating dynamic attributes
+    # of the WebLogic Domain
+    #useOnlineUpdate: true
```

Comment:
Shouldn't 'useOnlineUpdate' be under the 'model' stanza?  As far as I know it'll only be support for MII.

Reply:
@jshum2479 ^^^

Reply:
Still waiting for @rjeberhard  to have the schema finalized